scripts/import_map/update.ts

```diff
@@ -0,0 +1,13 @@
+import { parse } from "https://deno.land/std/flags/mod.ts";
```

Comment:
We should hardcode the version for this import.

nit: Should this come from `dev_deps.ts` as well?

Reply:
> Should this come from dev_deps.ts as well?

I dont feel like it should be part of `src/dev_deps.ts` but we could create a `script/dev_deps.ts` I'm just not sure if we want to treat the `script` folder as some sort of deno package

Reply:
Using `script/dev_deps.ts` does not feel like the proper approach since the `script` folder is not actually a module, the scripts are individually executing for the time being